app/uk/gov/hmrc/bindingtariffclassification/model/search/CaseParamsFilter.scala

```diff
@@ -16,5 +16,9 @@
 
 package uk.gov.hmrc.bindingtariffclassification.model.search
 
-case class CaseParamsFilter(queueId: Option[String] = None,
-                            assigneeId: Option[String] = None)
+case class CaseParamsFilter
+(
+  queueId: Option[String] = None,
+  assigneeId: Option[String] = None,
+  status: Option[Seq[String]] = None
```

Comment:
Do you think it is useful to use `Option[Seq[String]]` ?
How about defining?
```
status: Seq[String] = Seq.empty()
```

Reply:
Leaving this as it is, otherwise we need special logic for the empty list case when we convert it to a Mongo query